ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -2902,14 +2952,20 @@ const triggerSet: TriggerSet<Data> = {
           cn: '黑2: 踩第2轮白塔',
           ko: '긴 어둠 (두번째 빛 대상)',
         },
+        firstTower: {
+          en: 'First Tower',
+        },
+        secondTower: {
+          en: 'Second Tower',
+        },
```

Comment:
Do we need a merge version as well for the 0+2 strat?

Reply:
> Do we need a merge version as well for the 0+2 strat?

Sorry, not sure I understand what you're referring to by 'merge version.'  Do you mean the DNA slime making?  I don't think we call that out anywhere else in pangenesis.  Adding a "stack with partner" call for this or for towers would add more length to the text output or would get noisy if it's a second alert, so I'm hesitant.  (Also, tbh, I'm not entirely sure which sets of partners the slimes will come from - I think it can vary depending on who swaps?)

Reply:
Ah sorry, mostly saying rather than just saying "you're a 1, get second tower" should it say "you're a 1, get second tower but also merge first" (obviously not in those words).

Reply:
> Ah sorry, mostly saying rather than just saying "you're a 1, get second tower" should it say "you're a 1, get second tower but also merge first" (obviously not in those words).

Ohhhh, okay.  Yeah, sorry, I was being dense.  Done.